glide.yaml

```diff
@@ -34,6 +34,7 @@ import:
   - common/websocket
   - stream
 - package: github.com/uber/cherami-thrift
+  version: input_admin
```

Comment:
TODO: I will update this once the other PR is merged:
https://github.com/uber/cherami-thrift/pull/11